lib/tessel/reboot.js

```diff
@@ -0,0 +1,18 @@
+// System Objects
+// ...
+
+// Third Party Dependencies
+// ...
+
+// Internal
+var commands = require('./commands');
+var Tessel = require('./tessel');
+
+/*
+ Reboots Tessel
+ */
+Tessel.prototype.reboot = function() {
+  return new Promise((resolve,reject) => {
+    return this.simpleExec(commands.reboot()).then(resolve).catch(reject);
```

Comment:
Fixed. Promises are new to me so I appreciate comments like this. Also, discovered a bug! :)